src/rapids_dependency_file_generator/rapids_dependency_file_generator.py

```diff
@@ -390,4 +433,7 @@ def make_dependency_files(parsed_config, config_file_path, to_stdout):
                     os.makedirs(output_dir, exist_ok=True)
                     file_path = os.path.join(output_dir, full_file_name)
                     with open(file_path, "w") as f:
-                        f.write(contents)
+                        if file_type == str(OutputTypes.PYPROJECT):
+                            tomlkit.dump(contents, f)
```

Comment:
Would the `to_stdout` path also be split to use `tomlkit.dump()` ?

Reply:
Good question. The `to_stdout` code path doesn't really make sense for pyproject.toml files. The use case for conda env files is to generate new environments on demand, whereas pyproject.toml generation is only useful for a static file. There are a few code paths in here that will fail if `to_stdout == True` and any of the output types if `pyproject`. Rather than spending time trying to enable something that we don't need (at least not right now) I will add some error checking to ensure that this combination doesn't occur (the CLI won't permit it but someone could misuse the Python API in its current form.